lib/src/_shared/ToolbarButton.tsx

```diff
@@ -1,24 +1,31 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
 import clsx from 'clsx';
 import ToolbarText from './ToolbarText';
 import Button, { ButtonProps } from '@material-ui/core/Button';
 import { ExtendMui } from '../typings/extendMui';
+import { makeStyles } from '@material-ui/core/styles';
 import { TypographyProps } from '@material-ui/core/Typography';
-import { createStyles, withStyles, WithStyles } from '@material-ui/core/styles';
 
-export interface ToolbarButtonProps
-  extends ExtendMui<ButtonProps, 'variant'>,
-    WithStyles<typeof styles> {
+export interface ToolbarButtonProps extends ExtendMui<ButtonProps, 'variant'> {
   variant: TypographyProps['variant'];
   selected: boolean;
   label: string;
   align?: TypographyProps['align'];
   typographyClassName?: string;
 }
 
+export const useStyles = makeStyles(
+  _ => ({
```

Comment:
No theme?

```suggestion
  {
```

Reply:
Agree. 
But don't like such approach. Let me explain. From the perspective of changing styles from using theme or not. To change it you need 1) remove function 2) remove the `)` from the end of expression. 
I would like to use only 1 way to create theme. 
> Also for reference from the algebraic type system perspective we should avoid such polymorphic functions, cause they are giving more friction.

Reply:
Remembers me when we were switching between class components and functions components, eww!